src/Blocks/BlockDeadBush.h

```diff
@@ -42,6 +42,21 @@ class cBlockDeadBushHandler :
 		UNUSED(a_Meta);
 		return 0;
 	}
+
+	virtual void OnDestroyedByPlayer(cChunkInterface & a_ChunkInterface, cWorldInterface & a_WorldInterface, cPlayer * a_Player, int a_BlockX, int a_BlockY, int a_BlockZ) override
```

Comment:
Is this the best method to override?  [`BlockTallGrass` uses `DropBlock`](https://github.com/cuberite/cuberite/blob/5effe2cc7916e067e6f6579a73c1101a4e58dd63/src/Blocks/BlockTallGrass.h#L36).  That would also allow a plugin callback:

```a_BlockPluginInterface.CallHookBlockToPickups(a_Digger, a_BlockX, a_BlockY, a_BlockZ, m_BlockType, Meta, Drops);```